.github/workflows/build.yml

```diff
@@ -0,0 +1,95 @@
+name: Build app package
+
+on: [ push, pull_request, workflow_dispatch ]
+
+jobs:
+  build_pkg:
+    runs-on: ubuntu-20.04
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Checkout Dependecies
+      run: |
+        git clone https://github.com/illusion0001/apollo-lib.git -b no-dbglgr --depth 1
+        git clone https://github.com/bucanero/zip.git                         --depth 1
+        git clone https://github.com/bucanero/oosdk_libraries.git             --depth 1
+        git clone https://github.com/bucanero/mxml                            --depth 1
+        git clone https://github.com/Al-Azif/ps4-skeleton.git                 --depth 1
```

Comment:
Ideally these would use the checkout action.

Reply:
too many lines